tests/cim/new_test_cim_scenario.py

```diff
@@ -0,0 +1,384 @@
+import csv
+import os
+import pickle
+import tempfile
+import unittest
+from collections import namedtuple
+from typing import List, Optional
+
+import yaml
+
+from maro.simulator.scenarios.cim.business_engine import CimBusinessEngine
+
+os.environ["MARO_STREAMIT_ENABLED"] = "true"
```

Comment:
why true here?

Reply:
In order to enable `streamit` and test it. I set this environment variable according to [this doc](https://maro.readthedocs.io/en/latest/key_components/geographic_visualization.html).

Reply:
Sorry but where is the code to test the correctness of `streamit`?

Reply:
Currently there is no code to compare the expected output with the actual output of `streamit`. The "test" here means let `streamit` work and ensure it would not corrupted.

Reply:
OK